api/v1/swagger.yaml

```diff
@@ -7627,9 +8182,233 @@ definitions:
+      name:
+        type: "string"
+        description: "Should be unique."
+      currentGenerators:
+        readOnly: true
+        type: "array"
+        items:
+          type: "object"
+        description: "Generators assigned to this collector."
+      isRegistered:
+        type: "string"
+        description: "Boolean. If false, means de-registered. Changed by the DEREGISTER command"
+        enum:
+        - true
+        - false
+      status:
+        type: "string"
+        description: "The collector status."
+        enum:
+        - "STOPPED"
+        - "RUNNING"
+        - "PAUSED"
+      lastHeartBeat:
+        readOnly: true
+        type: string
+        description: >
+          Timestamp at which the last heartbeat was sent.
+      lastModifiedAt:
+        readOnly: true
+        type: string
+        description: >
+          Timestamp of the collector's most recent modification.
+      createdBy:
+        readOnly: true
+        type: string
+        description: >
+          Id of the User who created this Collector.
+      lastUpdatedBy:
+        readOnly: true
+        type: string
+        description: >
+          Timestamp of the collector's most recent update.
+      description:
+        type: string
+        maxLength: 4096
+        description: >
+          Describes this Collector; helps a user determine which Collectors they should assign to run their collection.
+      helpEmail:
+        type: string
+        description: >
+          The email address where a user can go to get more help about the collector.
+        maxLength: 254
+      helpUrl:
+        type: string
+        maxLength: 2082
+        description: >
+          The url where a user can go to get more help about the collector.
+
+  CollectorWriterResponse:
+    type: object
+    description: >
+      Users having write permission to the collector
+    properties:
+      collectorId:
+        type: string
+        readOnly: true
+        description: >
+          Id of a collector
+      userId:
+        type: string
+        readOnly: true
+        description: >
+          Id of a user having write permission to the collector
+      createdAt:
+        readOnly: true
+        type: string
+        description: >
+          Timestamp at which the the write permission was created
+      updatedAt:
+        readOnly: true
+        type: string
+        description: >
+          Timestamp at which the the write permission was modified
+
+  HeartbeatResponse:
+    type: object
+    description: >
+      Gets the config and sample collection metadata from the Refocus server.
+    properties:
+      collectorConfigUpdates:
+        $ref: "#/definitions/CollectorConfigUpdatesResponse"
+      generatersAdded:
+        $ref: "#/definitions/GeneratorsAddedResponse"
+      generatersModified:
+        $ref: "#/definitions/GeneratorsModifiedResponse"
+      generatorsRemoved:
+        $ref: "#/definitions/GeneratorsRemovedResponse"
+
```

Comment:
I think we want to have generatersUpdated and generatorsDeleted, instead of generatorsModified and generatorsDeleted.